frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -2037,6 +2038,122 @@ def encrypt(self, resulting_type: ConversionType, x: Conversion) -> Conversion:
     def equal(self, resulting_type: ConversionType, x: Conversion, y: Conversion) -> Conversion:
         return self.comparison(resulting_type, x, y, accept={Comparison.EQUAL})
 
+    def extract_bit(
```

Comment:
can't this call directly the more general case ? looks like this duplicate the exact same logic as the more general one

Reply:
The difference is the shifting for summation and early breaks.

Reply:
Actually I was thinking of removing `extract_bit` and using a simple slice of `0:1:1` instead in the generic one.